src/proxy.rs

```diff
@@ -143,18 +178,15 @@ impl ProxyHandlerHttp {
     ) -> ServerResult<Response<Body>> {
         // Construct the outbound URI & build a new request to be sent to the proxy backend.
         let outbound_uri = make_outbound_uri(&state.backend, req.uri())?;
-
-        let mut headers = req.headers().clone();
-        for (header_name, header_value) in state.request_headers.clone() {
-            if let Some(name) = header_name {
-                headers.insert(name, header_value);
-            }
-        }
-
-        let mut outbound_req = state
-            .client
-            .request(req.method().clone(), outbound_uri.to_string())
```

Comment:
The request method got lost in the refactor.

Reply:
Thanks for spotting this!